src/shared/breakpoint.js

```diff
@@ -0,0 +1,19 @@
+export default class Breakpoint {
+  static dataHero () {
```

Comment:
Are these values going to be related just to the hero? I think they are very generic for the project.